include/osquery/core.h

```diff
@@ -44,6 +44,9 @@ extern const std::string kSDKVersion;
 #define OSQUERY_VERSION STR(OSQUERY_BUILD_VERSION)
 #define OSQUERY_SDK_VERSION STR(OSQUERY_BUILD_SDK_VERSION)
 
+#define OSQUERY_BUILD_PLATFORM_DISTRO STR(OSQUERY_BUILD_OS)
```

Comment:
We can nix the `_DISTRO` part and keep the define strings the same as the defines. If we want to stick with the scheme above we can nix the `PLATFORM` part from the string defines. Or, we can use the `STR` macros within the `osquery_info` implementation.